tests/baselines/reference/declFileObjectLiteralWithOnlySetter.types

```diff
@@ -3,13 +3,13 @@
 === declFileObjectLiteralWithOnlySetter.ts ===
 function /*1*/makePoint(x: number) { 
 >makePoint : (x: number) => { b: number; x: number; }
->          : ^ ^^      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+>          : ^ ^^      ^^^^^^^^^^^^^^^^^^^^^      ^^^

```

Comment:
Why is this getting marked as reused? Presumably the return value is inside the function, so wouldn't it not be accessible?

Reply:
The way the syntactic node printer is now hooked up, it will get a chance to look for a type for any symbol whose types is serialized. So when printing the return type, the syntactic printer will be asked for a type for `x`, it then sees the accessor `set x` and the finds the `number` annotation, and since it's the same as the inferred type, it will be reused.